fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/MergeTwoProjectRule.java

```diff
@@ -34,7 +34,8 @@ public MergeTwoProjectRule() {
             resultMap.put(entry.getKey(), entry.getValue().accept(rewriter, null));
         }
 
-        OptExpression optExpression = new OptExpression(new LogicalProjectOperator(resultMap));
+        OptExpression optExpression = new OptExpression(
```

Comment:
Would better put this change to another PR, and merge it immediately.

Reply:
It's OK